### DISCUSSION TITLE
codeception module-phalcon5

### DISCUSSION THREAD

**Discussion by betrixed** [phalcon/phalcon#106] [Ideas]
I would like to be able to access, even read-only, the latest codeception module-phalcon5 , or whatever version or name it is, that is being used or shared for running tests on this current PHP evolving version. It appears to be available somewhere not known to myself.  Since the module-phalcon4, and module-phalcon all expect an installed C-phalcon extension, another module needs to be adapted for the evolution of this framework.  It should in theory , be possible to sieve through and adapt whatever old tests still apply from previous phalcon framework versions.

That would be great for me, because I pretty new to codeception, and it is a very nifty and hefty rig, and I want to get better at using it.
Some instructions on testing setup, in a prominent place, might encourage more contributions, since having run these tests for regression testing, and creating new tests compatible with the current framework as it evolves, as this is a mandatory requirement for contribution and testing. I could try and hack my own version, but this defeats the purpose of shared development and would be a waste of my time.

**Comment by Jeckerson** [comment]
Hello

https://github.com/Codeception/module-phalcon4

**Comment by malikwaqas758** [comment]
Hello
 just follow the link https://github.com/Codeception/module-phalcon4

**Comment by Arhell** [comment]
https://github.com/Codeception/module-phalcon5
